### DISCUSSION TITLE
Replacements deprecated APIs

### DISCUSSION THREAD

**Discussion by hliu1300** [ROCm/ROCm#2531] [Q&A]
Can ARM provide replacements for these deprecated APIs ?  
https://docs.amd.com/projects/HIP/en/latest/reference/deprecated_api_list.html

**Comment by hliu1300** [comment]
err: AMD

**Comment by darren-amd** [comment]
Hi @hliu1300,

Thanks for reporting this issue. The link you shared appears to be broken, but I found the depreciated API's here: [HIP Deprecated APIs](https://rocm.docs.amd.com/projects/HIP/en/latest/reference/deprecated_api_list.html). Would you mind sharing which API's you are interested in so I may assist further? Thanks!
